2996_reflection/reflection.md

```diff
@@ -4126,14 +4311,39 @@ Extend paragraph 5, and modify note 3, to clarify the existence of subobjects co
 
 :::
 
-### [over.pre] Preamble {-}
+Add a new paragraph to the end of the section defining _data member description_:
+
+::: std
+::: addu
+[29+]{.pnum} A _data member description_ is a quintuple (`$T$`, `$N$`, `$A$`, `$W$`, `$NU$`) describing the potential declaration of a nonstatic data member where
```

Comment:
```suggestion
[29+]{.pnum} A _data member description_ is a tuple (`$T$`, `$N$`, `$A$`, `$W$`, `$NU$`) describing the potential declaration of a non-static data member where
```

Reply:
Daveed pointed out that "quintuple" has precedence in the Standard:
https://eel.is/c++draft/over.built#9

Maybe we should keep as-is?